### PR TITLE
 On branch new_branch

### PR DESCRIPTION
**Pull request by kavehbarzegar**
new file:   temperature.py
adding a feature to the Project

### CODE REVIEW

**Comment by pouriya** [comment]
CI has failed!  
check the details and fix the issue please.

**Comment by kavehbarzegar** [comment]
I corrected mistakes

File: tabdil/temperature.py [hunk 1]
```diff
@@ -4,6 +4,7 @@ def fahrenheit(tem_unit):
     unit_tem = tem_unit[0].lower()
```
Comment (pouriya) [inline tabdil/temperature.py:4]: what is it for?

**Review by pouriya — CHANGES_REQUESTED** [review]

File: tabdil/temperature.py (hunk 1 above)
Comment (kavehbarzegar) [reply tabdil/temperature.py:4]: it helps to be sure unit_tem is lower case cause I used lower cases in my code

Comment (kavehbarzegar) [reply tabdil/temperature.py:4]: I fixed it, it should be deleted

**Comment by pouriya** [comment]
CI failed again!

**Comment by pouriya** [comment]
ping

**Pull request closed without merging**